content/jobs/healers/scholar/fight-tips/scholar-top.md

```diff
@@ -20,112 +20,140 @@ TOP requires less mana than DSR so you can just use this [2.40 set](https://etro
 * Fey Blessing is useful to hit everyone after towers
 * If mitigating Solars 3 and 4, I recommend placing Soil center as HWs go off if it’s not needed for the RDO hit
+* During the Dynamis mechanic of Sigma you can prep recite when you get to your spot for the Dynamis mechanic and when the far/close worlds go off you can Adlo yourself and spreadlo. This will help tanks for the buster if they need it and it will shield Run: Dyanmis(Omega).
+
 #### Run: Dynamis(Omega)
+
 * If mitigating this Solar Ray you can Reci Deploy off the tank taking the second hit, shielding the party for RDO and keeping the tank keeping autos healthy
 * Excogitation, Whispering Dawn, Fey Union, and Sacred Soil are all useful for keeping the tank taking autos healthy during the clone dodges
 * Try to Succor for each Hello World hit, but it shouldn’t be vital
-* Fey Blessing and Consolation again provide easy to use healing between Hello World sets
-* Preplace Soil to mit busters at the end of the mechanic, feel free to Adlo a tank at the end of the mechanic too
+* Fey Blessing and Consolation again provide easy-to-use healing between Hello World sets
+* You can pre-place Soil at the end of the Run: Dynamis(Omega) to mit busters at the end of the mechanic, feel free to Adlo a tank at the end of the mechanic too. You must do it near the end of Run: Dynamis(Omega) to make sure it's up for Blind Faith if you're using it for it.
 * Tanks that take the Blaster tethers will require healing since they’re set to 1% HP
 * Make sure to Succor for Blind Faith
 
 ### Phase Six: Alpha Omega
 
 #### General Soil tip
+
 * You can use macros for soil to help with brain space. It’s strongly recommended to use a target soil macro for Cosmo Meteors to get it directly middle allowing everyone to be inside of it. 
-* Do not use a target soil macro for Cosmo dive as your party can be clipped by the buster trying to stand inside the soil. You can manually place it like this image to make sure party and tanks are in it or you can place it on the party and trust your tanks to properly mit.
-* You can use a target macro for wc1 and a me macro for the rest of the mechanics if you want to use less brain space. 
+* Since the soil changes, using <t> macros for the entire phase leaves brain space room and it's very effective for every mechanic it's on.
+
 #### Cosmo Memory
+
 * Doesn’t need much mit to be survivable, I’d only use Soil here to help with the healing
 * Cosmo Dive 1
-* Preshield can be done when the second autoattack occurs after Cosmo Memory
+* Preshield can be done when the second auto-attack occurs after Cosmo Memory
 * Seraph can be used to cover both this and the first hits of Wave Cannon 1, and will then be back for Cosmo Meteor. The easiest way to time this is to Summon Seraph when Cosmo Dive begins casting and use the first charge immediately. Use the second charge when the second autoattack after Dive occurs
```

Comment:
An alternative use for seraph that's less execution heavy would be to use it to shield Cosmo Memory and then Cosmo Dive. Might be worth mentioning